pages/community/precice-workshop-organization.md

```diff
@@ -0,0 +1,116 @@
+- Overlap with other related conferences (check, e.g., ECCOMAS, IACM, SIAM)
+
+Once finalized, a "save the date" entry should be listed immediately on:
+
+- the preCICE website (news banner, community overview, first dummy page of the workshop)
+- the preCICE forum
+- the social media platforms that preCICE is active on
+
+### Find a venue
+
+If not at the same time as the date announcement, decide on a venue, and book rooms as soon as possible after that. Depending on the local workflows and availability, room booking requests might need to be done more than six months ahead, so aim to book at least the main room at the time that the venue is decided.
+
+Typical room requirements:
+
+- Fitting around 50 people (but not much larger than that)
+- For the hands-on training course and the user support sessions: easy access at every seat, (collaborative) working surface, enough power sockets, stable internet connection
+- For the talks: good acoustics and visibility, recording infrastructure
+- For the world café: (standing) tables, placed far from each other, isolated from noisy areas of the building
+- For the catering: ideally next to the rooms, isolated, standing
+
+With "recording infrastructure", we mean infrastructure to capture the speaker audio+video, slides, and an additional microphone in separate stream (a meeting camera system is typically not fitting the purpose). We can provide specialized hardware to do the mixing of these channels into what you later see as recordings on YouTube, if this is not already available in the room.
+
+The rooms for the training and the talks are typically different, even if the schedule is typically structured as a single track. The catering and the world café can typically use the same room and tables, but reusing the collaborative working room for the world café is also fine.
+
+### Define financing scheme
+
+Typically, the workshop is funded by participation fees. An important dependency for subsequent steps is which entity can collect and process such participation fees. This might be an association supporting scientific activities, an event management unit of a university, or the organizing institute (e.g., via project infrastructure).
+
+Multiple types of tickets are typically defined:
+
+- early -vs- late registration
+- full price -vs- discount for academia
+- training course included -vs- only main part of the workshop
+
+In addition, free registration is accounted for:
+
+- holders of a support license
+- invited speakers
+- special cases
+
+### Set milestones
+
+Set clear responsible persons for each task, and track progress in a collaborative project board.
+
+Important dates to set:
+
+- "save the date" announcement: one year ahead
+- call for contributions (needs a form): open six months ahead. An early call mainly acts as advertisement.
+- open registration (needs a form): six months ahead. Early registration is important for participants that need a visa application to travel.
```

Comment:
Sounds too early to me. Of course, early does not hurt. Visa application should only depend on the contribution, not on the registration, no?